src/main/java/seedu/hirehub/logic/commands/SearchApplicationCommand.java

```diff
@@ -20,6 +20,8 @@ public class SearchApplicationCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all applications whose attributes match all "
             + "the corresponding specified attributes.\n"
+            + "For email and status, only full words will be matched,\n"
+            + "while for title, partial words will be matched.\n"
```

Comment:
possible violation of feature freeze

Reply:
Indeed, as this seems to be making the message usage more specific. Will revert, thanks!